### PR TITLE
feat(dashboard,types): add active order change panel - claims, exchanges & returns

### PR DESCRIPTION
**Pull request by riqwan**
what:

- adds an active panel for order changes - claims, exchanges & returns

Screenshot 2024-08-23 at 13 08 45

### CODE REVIEW

File: packages/admin-next/dashboard/src/i18n/translations/en.json [hunk 1]
```diff
@@ -912,6 +912,14 @@
           "errorNegativeValue": "Quantity cannot be a negative value.",
           "errorLargeDamagedValue": "Damaged items quantity + non damaged received quantity exceeds total item quantity on the return. Please decrease quantity of non damaged items."
```
Comment (olivermrbl) [inline packages/admin-next/dashboard/src/i18n/translations/en.json:913]: nit(this-file): Should be `canceled` with a single `l`.

**Review by carlos-r-l-rodrigues — APPROVED** [review]

**Review by olivermrbl — APPROVED** [review]

File: packages/admin-next/dashboard/src/i18n/translations/en.json [hunk 2]
```diff
@@ -912,6 +912,14 @@
...
         }
+      },
+      "toast": {
+        "canceledSuccessfully": "Return cancelled successfully",
```
Comment (riqwan) [inline packages/admin-next/dashboard/src/i18n/translations/en.json]:
```suggestion
        "canceledSuccessfully": "Return canceled successfully",
```

File: packages/admin-next/dashboard/src/i18n/translations/en.json [hunk 3]
```diff
@@ -931,6 +939,14 @@
       },
       "tooltips": {
         "onlyReturnShippingOptions": "This list will consist of only return shipping options."
+      },
+      "toast": {
+        "canceledSuccessfully": "Claim cancelled successfully",
```
Comment (riqwan) [inline packages/admin-next/dashboard/src/i18n/translations/en.json]:
```suggestion
        "canceledSuccessfully": "Claim canceled successfully",
```

File: packages/admin-next/dashboard/src/i18n/translations/en.json [hunk 4]
```diff
@@ -950,6 +966,14 @@
       },
       "tooltips": {
         "onlyReturnShippingOptions": "This list will consist of only return shipping options."
+      },
+      "toast": {
+        "canceledSuccessfully": "Exchange cancelled successfully",
```
Comment (riqwan) [inline packages/admin-next/dashboard/src/i18n/translations/en.json]:
```suggestion
        "canceledSuccessfully": "Exchange canceled successfully",
```

**Pull request merged**